### PR TITLE
[openmvs] update to v1.1

### PR DESCRIPTION
**Pull request by cdcseacave**

### CODE REVIEW

**Comment by cdcseacave** [comment]
hmmm, I'm lost, the logs say this
```
       "C:\vsts\_work\1\s\buildtrees\osg\x64-windows-dbg\examples\osgmovie\example_osgmovie.vcxproj" (default target) (88) ->
       (ClCompile target) -> 
         C:\vsts\_work\1\s\buildtrees\osg\src\raph-3.6.4-9780889fab\examples\osgmovie\osgmovie.cpp(694,10): error C1083:  Cannot open include file: 'SDL/SDL.h': No such file or directory [C:\vsts\_work\1\s\buildtrees\osg\x64-windows-dbg\examples\osgmovie\example_osgmovie.vcxproj]
```
however I compile *OpenMVS* which does not use or contain *osg* (do not even know what that is. So how come it does that and why it repots failure for my port?

**Review by PhoebeHui — APPROVED** [review]
LGTM

**Comment by cdcseacave** [comment]
?

File: ports/openmvs/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: openmvs
-Version: 1.0.1
+Version: 1.1.0
```
Comment (PhoebeHui) [inline ports/openmvs/CONTROL]: Could you update the Version to '1.1'?

**Review by PhoebeHui — CHANGES_REQUESTED** [review]

File: ports/openmvs/CONTROL (hunk 1 above)
Comment (cdcseacave) [reply ports/openmvs/CONTROL]: done, thx

**Review by PhoebeHui — APPROVED** [review]

**Comment by strega-nil** [comment]
Thanks @cdcseacave :)

**Pull request merged**